server/routes/auth.py

```
@@ -495,6 +536,73 @@ def post(self):
+
+            threading.Thread(
+                target=send_invitation_email,
+                args=(user.email, user.full_name, verify_link, "System", None),
+                daemon=True,
+            ).start()
+
+            return {
+                "status": "success",
+                "message": "Verification link resent successfully",
+                "data": None,
+            }, 200
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)